src/test/java/org/apache/commons/io/FileUtilsCopyDirectoryToDirectoryTestCase.java

```diff
@@ -73,6 +83,48 @@ public void copyDirectoryToDirectoryThrowsNullPointerExceptionWithCorrectMessage
         assertExceptionTypeAndMessage(srcDir, destDir, NullPointerException.class, "destinationDir");
     }
 
+
+    @Test
+    public void copyFileWrongPermissions() throws IOException {
+        
+    	 
+        final File destDir = createTemporaryFolderIn(null);
+        final  Path srcFile = Files.createTempFile("tmp-output", ".xml");
+        final Path path = Paths.get(destDir.getAbsolutePath(), "newFile.xml");
+
+        try {
+            FileUtils.copyFile(srcFile.toFile(), path.toFile());
+        } catch (IllegalArgumentException iae) {
+        	iae.printStackTrace();
+        }
+        
+        assertTrue(Files.getPosixFilePermissions(path).contains(PosixFilePermission.OTHERS_READ), Files.getPosixFilePermissions(path).toString());
```

Comment:
@cosmoJFH 
Here you are testing that a target path has a certain attribute but you are not checking that this attribute was in fact present in the source.